### PR TITLE
feat :  done with questions page

### PR DESCRIPTION
**Pull request by iprankurpandey**

### CODE REVIEW

File: questions.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    background-color: purple;
+    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.75);
+    margin: 1rem;
+    cursor: pointer;
+    font-weight: bold;
+}
+
+.brand-choice {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
```
Comment (Sahil25061999) [inline questions.css:47]: there is no need for repeat here.

File: questions.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    height: auto;
+
+}
+
+.options{
+    /* padding: 1rem;
+    margin: 1rem;
+    font-size: 1rem;
+    cursor: pointer; */
+}
```
Comment (Sahil25061999) [inline questions.css:82]: remove this if not necessary

File: questions.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    width: 100%
+}
+
+.social-media-links {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    height: 2rem;
+    background-color: silver;
```
Comment (Sahil25061999) [inline questions.css:128]: a lot of code is being repeated just use " , " separator and give the properties to both the classed together

**Review by Sahil25061999 — COMMENTED** [review]
I think having the login button on the right will be better rather than having it centered, also add hover effect when selecting an answer, same goes for button too.

**Review by manish-GH — COMMENTED** [review]
The login button looks weird in the center, probably you could move it to the right.

From what I understand we are not supposed to have all the questions on one screen. We have to display the 2nd question after the 1st question has been answered.

Also, I don't think all the colors go together, you could probably change your colors.

**Pull request merged**